CITATION.cff

```diff
@@ -77,8 +77,23 @@ authors:
 - family-names: "Graham"
   given-names: "Brett"
   orcid: "https://orcid.org/0000-0001-6315-4507"
+- family-names: "Molter"
+  given-names: "Ned"
```

Comment:
@emolter - would you prefer to be cited as Ned or Edward?

Reply:
I suppose Edward since that's the name I publish with.  It doesn't matter too much though since my ORCID is there.

Reply:
Thanks for checking!